english/krishna/glories

```diff
@@ -0,0 +1 @@
+Click [here] (http://backtogodhead.in/all-glories-to-lord-sri-krishna/) to visit glories of krishna
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.